edx_lint/cmd/write.py

```diff
@@ -82,6 +83,8 @@
 # SERIOUSLY.
 #
 # ------------------------------
+# VERSION: {version}
```

Comment:
nit: this is the version of edx-lint that produced the file, not the version of the file.  The written files in each repo can vary from each other because of local tweaks files.

Reply:
That's intentional, but I'll make it clearer in the tag. I want to have the version that generated the file so that we can validate that all repos are using the latest generated settings.